packages/storage/src/providers/s3/utils/isPresignedUrlDurable.ts

```diff
@@ -0,0 +1,25 @@
+import { extendedEncodeURIComponent } from '@aws-amplify/core/internals/aws-client-utils';
+
+export function isPresignedURLDurable(
```

Comment:
It doesn't have to be called `isPresignedURLDurable`, because the URLs we're checking aren't necessarily signed yet. Maybe something like `validateObjectUrl`?

Reply:
Updated